oauth_dropins/reddit.py

```diff
@@ -137,19 +137,21 @@ def praw_to_user(user):
   Note 1: accessing redditor attributes lazily calls reddit API
   Note 2: if user.is_suspended is True, other attributes will not exist
   Note 3: subreddit refers to a user profile (stored as a subreddit)
+  Ref: https://praw.readthedocs.io/en/latest/code_overview/models/redditor.html
 
   Returns: dict
 
   Raises:
     :class:`prawcore.exceptions.NotFound` if the user doesn't exist or has been
     deleted
   """
-  if user.is_suspended:
+  if getattr(user, 'is_suspended', False):
+    return {}
+  if any([not hasattr(user,a) for a in ['name','subreddit','icon_img','id','created_utc']]):
```

Comment:
instead of requiring them all like this, let's maybe use `getattr` inline below? eg:

```py
return {
  'name': getattr(user, 'name', None),
  ...
```

that way we return anything we do get.